speedy-recovery-client/src/components/secured/shared/Header.js

```diff
@@ -1,29 +1,33 @@
 import React, { Component } from "react";
 import { Link } from "react-router-dom";
-import { Container, Dropdown, Icon, Image, Label, Menu } from "semantic-ui-react";
+import {Container, Dropdown, Icon, Image, Label, Menu} from "semantic-ui-react";
 import "./Header.css";
 
 class Header extends Component {
   render() {
+    let showMessage;
+    if(this.props.role !== "Patient"){
+       showMessage = <Menu.Item as={Link} to="/secured/messaging" >
+       <Icon name="mail" />
+       Messages
+       <Label color="teal" circular>
```

Comment:
Is there not a way to update this based on the number of new messages?

Reply:
for now, we directly get the "unread" numbers from the backend, so that backend need to add some function to update the "unread" number, and then we can display it in frontend, but there are a lot of stuffs to change, so I prefer to finish the main feature (practitioner and parent info view) first, if we have enough time, we will try to fix it.

Reply:
I am looking into it